docs/source/changelog.rst

```diff
@@ -6,13 +6,14 @@ Changelog
     * Enhancements
     * Fixes
     * Changes
+        * Deprecated ``feature's get_names`` function, ``get_feature_names`` should be used instead
```

Comment:
Maybe change these to `Feature.get_names()` and `Feature.get_feature_names()` instead.